src/app/shared/components/policy-link/policy-link.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { Params, Router } from '@angular/router';
+import { StoreUrlService } from '@app/core/store-url/store-url.service';
+import { CategoryRoutingName, CorePath } from '@app/shared/models';
+import { LanguageCode } from '@app/core/translations';
+import { LocalStorageKeys } from '@app/shared/models';
+
+@Component({
+  selector: 'app-policy-link',
+  templateUrl: './policy-link.component.html',
+  styleUrls: ['./policy-link.component.scss'],
+})
+export class PolicyLinkComponent {
+  corePath = CorePath;
+
+  // TODO: introduce grammatical cases, starting with denominator and accusative.
+  @Input() denominator: boolean = true;
```

Comment:
Be careful with generalizing it; different languages may have different grammar cases and different rules when to use which. In general, we should instead start using proper translation tools; this is just a hack for MVP1.

Reply:
Ack. +1. I propose not to address in this PR.